src/Symfony/Component/Messenger/Asynchronous/Middleware/SendMessageMiddleware.php

```diff
@@ -34,14 +34,15 @@ public function __construct(SenderLocatorInterface $senderLocator, array $messag
     }
 
     /**
+     * @param Envelope $envelope
```

Comment:
Since the component is experimental, we can turn this to a real type hint on master at least?

Reply:
There is a PR for this (https://github.com/symfony/symfony/pull/27322) but I don't think that it's a good idea. At all.